cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/tableSegment/PutTableSegmentEntryCommand.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.segmentstore.tableSegment;
+
+import io.pravega.cli.admin.CommandArgs;
+import io.pravega.cli.admin.utils.AdminSegmentHelper;
+import io.pravega.client.tables.impl.TableSegmentEntry;
+import io.pravega.client.tables.impl.TableSegmentKeyVersion;
+import io.pravega.common.util.ByteArraySegment;
+import io.pravega.shared.protocol.netty.PravegaNodeUri;
+import lombok.Cleanup;
+import org.apache.curator.framework.CuratorFramework;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public class PutTableSegmentEntryCommand extends TableSegmentCommand {
+
+    /**
+     * Creates a new instance of the PutTableSegmentEntryCommand.
+     *
+     * @param args The arguments for the command.
+     */
+    public PutTableSegmentEntryCommand(CommandArgs args) {
+        super(args);
+    }
+
+    @Override
+    public void execute() {
+        ensureArgCount(4);
+        ensureSerializersExist();
+
+        final String fullyQualifiedTableSegmentName = getArg(0);
+        final String segmentStoreHost = getArg(1);
+        final String key = getArg(2);
+        final String value = getArg(3);
```

Comment:
As a user - How do I provide binary value?

Reply:
So the value is provided as a special string with key-value pairs in the form `key1=value1;key2=value2;.....`. The serializers read and parse this special string to generate the object which is then serialized into binary and then sent in the `updateTableEntries` call.